docs/adfs/README.md

```diff
@@ -111,6 +111,16 @@ module.exports = passport;
 ```
 # Troubleshooting
 
+## Errors you might encounter
+
+|Error   | Explanation  | Mitigation  |
+|--------|---|---|
+| `SAML assertion not yet valid`  | This means that there is a clock skew between your server time and your saml provider time. There might be a better way by fiddling with the ClockSkew You can disable clock skew validation with a value of `-1`.  | `{ acceptedClockSkewMs: -1 }`  |
+| `SAML provider returned Requester error: InvalidNameIDPolicy`  | Setting the   |   |
```

Comment:
Is there a reference in official SAML docs about the use the Name ID policies that can be linked to? Again, just disabling a policy that's causing a problem doesn't seem like the best solution.